core/settings.py

```diff
@@ -0,0 +1,128 @@
+"""
+Django settings for core project.
+
+Generated by 'django-admin startproject' using Django 5.1.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/5.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/5.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = (
+    "django-insecure-g+3o3xlz=om-yvrbg*&uskzpsiri*-o9ufc%xvv#1#2+w6pb=i"

```

Comment:
The `SECRET_KEY` is currently hardcoded, which is acceptable for development. However, for production, it's recommended to use environment variables or a secrets management tool to keep it secure.